src/nginx/etc/nginx/nginx.conf

```diff
@@ -23,6 +23,8 @@ http {
 
     keepalive_timeout 65;
 
+    client_max_body_size 6m;
```

Comment:
I don't have further insight into how big this should be. A few megabytes as modified here is definitely sufficient for now.